Tests/SwiftProtobufTests/Test_BinaryDelimited.swift

```diff
@@ -15,6 +15,7 @@ import SwiftProtobuf
 class Test_BinaryDelimited: XCTestCase {
 
   func testEverything() {
+#if swift(>=3.1)
```

Comment:
`OutputStream.toMemory` and `XCTAssertNoThrow` are available only starting in Swift 3.1 (on Linux, at least).  This is the only test that uses those, so I've just commented out the whole thing.  We could work around those missing functions if you think it's important to run this test on older Swift.